app/uploaders/lighthouse_document_uploader.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Files that will be associated with a previously submitted claim, from the Claim Status tool
+class LighthouseDocumentUploader < LighthouseDocumentUploaderBase
+  def initialize(icn, ids)
+    # carrierwave allows only 2 arguments, which they will pass onto
+    # different versions by calling the initialize function again,
+    # that's why i put all ids in the 2nd argument instead of adding a 3rd argument
+    super
+    @icn = icn
+    @ids = ids
+    set_storage_options!
+  end
+
+  def store_dir
+    store_dir = "lighthouse_documents/#{@icn}"
+    @ids.compact.each do |id|
+      store_dir += "/#{id}"
+    end
+    store_dir
+  end
+
+  def move_to_cache
+    false
+  end
+
+  private
+
+  def set_storage_options!
```

Comment:
This does not exist in settings. I copied it over from EVSS and also updated the settings in S3, but I'm not sure if we're supposed to be directly uploading to S3, or if the endpoint is supposed to receive the full payload